install/kubernetes/cilium/charts/operator/values.yaml

```diff
@@ -1,4 +1,8 @@
-image: operator
+image:
+  repository: operator
+  tag: latest
+  imagePullPolicy: Always
```

Comment:
related to above comment, this field can be rename to just _pullPolicy_ (as .Values.image.imagePullPolicy looks a little bit weird I think).